src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
You don't need to create a Lottery instance every time you want to get a random ball. Instead, you can reuse an existing instance. In this case, it won't affect performance or memory in any way, but on real-world projects, unnecessarily creating a heavy object can cause problems.
I would also recommend separating System.our.println and the object creation logic with an empty line.

```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
        Lottery lottery = new Lottery(colorSupplier);

        System.out.println(lottery.getRandomBall().toString());
        System.out.println(lottery.getRandomBall().toString());
        System.out.println(lottery.getRandomBall().toString());
```